docs/operators/server.md

```diff
@@ -227,6 +227,14 @@ schema @server(
 )
 ```
 
+## showcase
+
+Enables the `/showcase/graphql` endpoint. This allows anyone to run arbitrary GraphQL schemas by passing a URL in the `config` query param.
+
+```graphql showLineNumbers
+schema @server(showcase: true)
+```
+
```

Comment:
Can you write a bit more about this feature? I don't think how one could use this feature is very clear from this note.